tests/show_results.py

```diff
@@ -52,6 +54,29 @@ def translate_path(self, path: str) -> str:
             path += "/"
         return path
 
+    def do_POST(self) -> None:
+        if self.path == "/fixtures.json" and os.path.exists(FIXTURES_PATH):
+
+            length = int(self.headers.get("content-length"))
+            field_data = self.rfile.read(length)
+            data = json.loads(field_data)
+
+            with open(FIXTURES_PATH, "r") as jsonFile:
+                fixtures = json.load(jsonFile)
+
+            test_name = data.get("test", None)
```

Comment:
`None` explicit default value is theoretically not needed, `get(key)` has `None` default by default.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/a2900711241302cb3b2b7f4d1cac0f538ecf4960